### ISSUE TITLE
Issues/Feedback on pm.gc.ca

### ISSUE THREAD

**Issue by thekodester** [wet-boew/GCWeb#1560]
Not sure who manages https://pm.gc.ca/eng, but the useless scroll effects cause flickering on Safari with macOS 10.14.5.

Also from a UI point of view the site feels cluttered and unorganized (not very professional).

**Comment by RobJohnston** [comment]
I used to work a bit on that site about 4 years ago and it was done by Privy Council Office at the time.  They may be except from strictly following WET.  Paging @jamessauve?

**Comment by thekodester** [comment]
Yeah the from is not even following WET, but the usability of the site is now not great. I'm all for innovative designs as long as there user friendly, accessible, and don't cause glitching.

**Comment by thekodester** [comment]
Here a GIF of the issue (you can see the major jitteriness of the page when scrolling):
![glitch](https://user-images.githubusercontent.com/23484460/60473133-b4173700-9c39-11e9-9013-8c7829686a39.gif)

**Comment by thekodester** [comment]
Screenshot of the cluttered UI (mostly the header is too big on smaller screen and twitter feed takes up a lot of room): 
<img width="1388" alt="Screen Shot 2019-07-01 at 7 54 02 PM" src="https://user-images.githubusercontent.com/23484460/60473282-62bb7780-9c3a-11e9-8ef6-e4d8be6390e4.png">

**Comment by thekodester** [comment]
Also the type of computer I use is a MacBook Pro (Retina, 13-inch, Early 2015). 13-inch laptops are quite common these days so it a shame they didn't make sure the site looked okay on smaller screens, but I get it cause a lot of developers work with large monitors.

**Comment by GormFrank** [comment]
Does anyone have updates on that? @duboisp did you get feedback since you passed on the info?
